### PR TITLE
[12.0][MIG] partner_default_sale_discount

### PR DESCRIPTION
**Pull request by lbelorgey**
Standard migration of module partner_default_sale_discount.

### CODE REVIEW

File: partner_default_sale_discount/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_default_sale_discount/__init__.py]: Please change license lines in this module to https:

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: LGTM but would be could to change license lines to https: to make it complete up to date with conventions.

File: partner_default_sale_discount/__init__.py (hunk 1 above)
Comment (lbelorgey) [reply partner_default_sale_discount/__init__.py]: Done.

**Review by NL66278 — APPROVED** [review]
:+1: Thanks for the migration!

**Comment by lbelorgey** [comment]
Travis is now red with a strange error:
`fatal: unable to access 'https://github.com/OCA/maintainer-quality-tools.git/': Failed to connect to github.com port 443: Connection timed out`

What can I do ?

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Pull request merged**